src/ex1/style.css

```diff
@@ -0,0 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+body {
+    width: 100%;
+    height: 100vh;
```

Comment:
in order for the viewport styles (vh/vw) to apply correctly you need to add this meta tag to the HTML header <meta name="viewport" content="width=device-width, initial-scale=1">

You can read more on css viewport here https://www.educba.com/css-viewport/